### ISSUE TITLE
app-os-monitor build failed

### ISSUE THREAD

**Issue by tsh2** [me-box/databox#245]
I built the app-os-monitor in hub.docker,but failed 
![ee1](https://user-images.githubusercontent.com/36950432/37505545-da7bdaa2-2928-11e8-810e-52450df9af02.png)

And I also try the docker build code in my os,same error occured 

![tocon1](https://user-images.githubusercontent.com/36950432/37505608-16b724e0-2929-11e8-940d-117513a30c51.png)

And when run  databox-start dev,it also error occured in app-os-monitor same error.

**Comment by Toshbrown** [comment]
In the docker file 

RUN go get -u github.com/toshbrown/lib-go-databox 

should be 

 RUN go get -u github.com/me-box/lib-go-databox

**Comment by tsh2** [comment]
thanks you, I solved it.
When I run with dev mode,The login page is appeared.
Is there any Databox user control setting?

![tocon1](https://user-images.githubusercontent.com/36950432/37577609-7fd08562-2b76-11e8-9607-21a711cd3714.png)

When  I used the following password ,I can login.

![tocon2](https://user-images.githubusercontent.com/36950432/37577761-6b86969a-2b77-11e8-9640-f047771e0589.png)

And then ,In app-os-monitor ,what kind of data are exported?

![tocon3](https://user-images.githubusercontent.com/36950432/37577947-8ae7b52c-2b78-11e8-85b3-7d848ce0834b.png)
